corporate/tests/test_activity_views.py

```diff
@@ -367,6 +367,6 @@ def add_audit_log_data(
         add_audit_log_data(realm.server, remote_realm=realm, realm_id=None)
 
         self.login("iago")
-        with self.assert_database_query_count(12):
+        with self.assert_database_query_count(11):
```

Comment:
One less database query for fixed price plans in these views, as we're no longer calling `get_next_billing_cycle` in `get_customer_renewal_amount` for these plans.